src/test/runTest.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+
+import * as glob from "glob";
+import { runTests } from "@vscode/test-electron";
+
+async function main() {
+  try {
+    // The folder containing the Extension Manifest package.json
+    // Passed to `--extensionDevelopmentPath`
+    const extensionDevelopmentPath = path.resolve(__dirname, "../../");
+
+    // The path to the extension test script
+    // Passed to --extensionTestsPath
+    const extensionTestsPath = path.resolve(__dirname, "./suite/index");
+
+    const vscodePaths = glob.sync(
+      `${extensionDevelopmentPath}/.vscode-test/vscode-*/code`
+    );
+
+    if (vscodePaths.length == 0) {
+      throw new Error("Unable to find vscode executable");
+    }
+
+    const vscodeExecutablePath = vscodePaths[0];
+
+    // Download VS Code, unzip it and run the integration test
+    await runTests({
+      extensionDevelopmentPath,
+      extensionTestsPath,
+      vscodeExecutablePath,
+    });
+  } catch (err) {
+    console.error("Failed to run tests\n" + err);
+    process.exit(1);
+  }
+}
+
+main();
```

Comment:
Did you make the main function just to use async?

In newer Node you can use async/await without a function.

Reply:
In my editor and when trying to run the test I receive the following error (running with node 16.14.2):

```
src/test/runTest.ts:25:3 - error TS1378: Top-level 'await' expressions are only allowed when the 'module' option is set to 'es2022', 'esnext', 'system', 'node16', or 'nodenext', and the 'target' option is set to 'es2017' or higher.
```